robotpy_build/platforms.py

```diff
@@ -51,6 +50,20 @@ def get_platform() -> WPILibMavenPlatform:
     #       be useful to note for the future.
 
     pyplatform = _get_platform()
+
+    if pyplatform == "linux-armv7l":
+        try:
+            import distro
+
+            if distro.id() == "nilrt":
```

Comment:
I don't really like this, but I don't have a good alternative.

Reply:
Nor do I honestly. Any alternatives I could come up with will probably be worse.

At least this is mockable to some extent.